io.github.Fndroid.clash_for_windows.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
Ideally this should be upstreamed.

Reply:
> Ideally this should be upstreamed.

Hi, I have successfully located and solved the problem with [this commit](https://github.com/qvshuo/flathub/commit/210cc7e9df55d99410d04e78921a1c05941fbd53) thank you very much for your advice.

Reply:
It looks like upstream may not be paying attention to flathub or even LInux, which is a sad story.